frontend/app/src/components/asset-manager/RestoreAssetDbButton.vue

```diff
@@ -161,4 +126,42 @@ async function updateComplete() {
   await restartBackend();
   await connect();
 }
+
+const { show } = useConfirmStore();
+
+const showRestoreConfirmation = (type: ResetType) => {
+  set(resetType, type);
```

Comment:
Same here, we can do the same thing I mentioned above, we can remove the `resetType` and pass the id bellow to the method.